react/features/etherpad/components/native/SharedDocument.js

```diff
@@ -71,20 +75,28 @@ class SharedDocument extends PureComponent<Props> {
      */
     render() {
         const { _documentUrl } = this.props;
+        const { navigation } = this.props;
+
+        navigation.setOptions({
+            headerLeft: () => (
+                <TouchableWithoutFeedback
+                    /* eslint-disable-next-line react/jsx-no-bind */
+                    onPress = { () => navigation.goBack() } >
+                    <Icon
+                        size = { 20 }
+                        src = { IconArrowBack }
+                        /* eslint-disable-next-line react-native/no-inline-styles */
+                        style = {{ marginLeft: 12 }} />
+                </TouchableWithoutFeedback>
+            )
+        });
 
         return (
-            <JitsiModal
-                headerProps = {{
-                    headerLabelKey: 'documentSharing.title'
-                }}
-                modalId = { SHARE_DOCUMENT_VIEW_ID }
-                style = { styles.webView }>
-                <WebView
-                    onError = { this._onError }
-                    renderLoading = { this._renderLoading }
-                    source = {{ uri: _documentUrl }}
-                    startInLoadingState = { true } />
-            </JitsiModal>
+            <WebView
```

Comment:
Shouldn't this component be surrounded by a JitsiScreen ?

Reply:
Have you tested with an iPhone with a notch? What the modal wrapper (and now the screen wrapper) gives us is the safe area and keyboard avoiding view.

Reply:
Managed to wrap them all inside JitsiScreens.